### PR TITLE
docs: setup issue template

### PR DESCRIPTION
**Pull request by huozhi**
* create issue template for bug reporting
* add external link to swr discussion board

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Create a bug report for the Next.js core / examples
+---
+
+# Bug report
+
+## Description / Observed Behavior
+
+what kind of issues did you encounter with swr?
```
Comment (shuding) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]:
```suggestion
What kind of issues did you encounter with SWR?
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Expected Behavior
+
+what did you expect swr to behave here?
```
Comment (shuding) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]:
```suggestion
What did you expect SWR to behave here?
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## Repro Steps / Code Example
+
+Or share your code snippet or a codesandbox link is also appreciated!
```
Comment (shuding) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]:
```suggestion
Or share your code snippet or a [CodeSandbox](https://codesandbox.io) link is also appreciated!
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+## Additional Context
+
+swr version.
```
Comment (shuding) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]:
```suggestion
SWR version. 
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Create a bug report for the Next.js core / examples
```
Comment (pacocoursey) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]:
```suggestion
about: Create a bug report for the SWR library
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (pacocoursey) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]:
```suggestion
How did you expect SWR to behave here?
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 5 above)
Comment (shuding) [reply .github/ISSUE_TEMPLATE/bug_report.md:3]: Great great catch!

Comment (huozhi) [reply .github/ISSUE_TEMPLATE/bug_report.md]: now you know this line is from next.js repo 🙈

**Pull request merged**